### DISCUSSION TITLE
Am I able to build a GUI app which can display lines for a 3D modelling software?

### DISCUSSION THREAD

**Discussion by Joe23232** [emilk/egui#285] [General]
You see these lines in a 3D modelling software:

![image](https://user-images.githubusercontent.com/34926497/114023921-cadec900-98b6-11eb-9d8f-9cafad57c83a.png)

Am I able to make those kinds of lines using this software?

Additionally am I able to render a scene from something such as the [Bevy Game engine](https://bevyengine.org/) and then display it and then on the side I can add buttons using this `egui` crate?

For example:

![image](https://user-images.githubusercontent.com/34926497/114024448-6b34ed80-98b7-11eb-93c7-3a38f9757fc3.png)

**Comment by emilk** [comment]
3D rendering is best done with a 3D library, and yes you can use bevy with egui: https://docs.rs/bevy_egui/0.3.0/bevy_egui/

**Comment by Joe23232** [comment]
Awesome thanks :)
